workos/src/main/kotlin/com/workos/portal/models/Intent.kt

```diff
@@ -2,7 +2,19 @@ package com.workos.portal.models
 
 import com.fasterxml.jackson.annotation.JsonValue
 
+/**
+ * Intent of the Admin Portal.
+ *
+ * @param value The string value of the intent.
```

Comment:
Should intent be capitalized in this context?
```suggestion
 * @param value The string value of the Intent.
```